Add note about changing SQL Trait data type (#5620)
segmentio/segment-docs: src/unify/Traits/sql-traits.md

> info ""
> To view SQL Traits in a user profile, you must have [PII access](/docs/segment-app/iam/roles/#pii-access). Without this access, Segment redacts all SQL traits in a profile.

> info ""

> > Once a field has been brought in with a SQL Trait and sent to the destination, please note that changing data types for fields may not be compatible with all destinations.

Reviewer: ```suggestion
> Note that after you bring in data with SQL Traits, changing data types for fields may not be compatible with all destinations.
```
